### PR TITLE
Chore: Adjust preview image

### PR DESCRIPTION
**Pull request by marianaviana**
__________________________________________________________
### Issue https://github.com/universoimpulso/comunidadestech/issues/148#issue-606466038
__________________________________________________________
seo-ctech

### CODE REVIEW

File: pages/_document.js [hunk 1]
```diff
@@ -104,20 +103,51 @@ export default class MyDocument extends Document {
...
+          <link
+            rel="manifest"
+            href="/static/manifest.json"
+          />
+          <script
+            src="/static/sw-register.js"
+          ></script>
           <script
             async
             src="https://www.googletagmanager.com/gtag/js?id=UA-143000900-1"
```
Comment (rtarantelli) [inline pages/_document.js:143]: Can this URL be a variable?

File: pages/_document.js [hunk 2]
```diff
@@ -104,20 +103,51 @@ export default class MyDocument extends Document {
...
           ></script>
-          <script dangerouslySetInnerHTML={this.setGoogleTags()} />
-          <script src="https://kit.fontawesome.com/e258bd240c.js"></script>
+          <script
+            dangerouslySetInnerHTML={this.setGoogleTags()}
+          />
+          <script
+            src="https://kit.fontawesome.com/e258bd240c.js"
```
Comment (rtarantelli) [inline pages/_document.js:149]: Can this URL be a variable?

File: pages/_document.js (hunk 1 above)
Comment (marinhoviana) [reply pages/_document.js:143]: Not in that case.

File: pages/_document.js (hunk 2 above)
Comment (marinhoviana) [reply pages/_document.js:149]: In that case, it is also not necessary.

**Review by omarciovsena — APPROVED** [review]

File: pages/_document.js (hunk 2 above)
Comment (rtarantelli) [reply pages/_document.js:149]: Why is it not necessary? You can explain better?

File: pages/_document.js (hunk 1 above)
Comment (rtarantelli) [reply pages/_document.js:143]: Why is it not necessary? You can explain better?

**Pull request merged**